config/config.go

```diff
@@ -66,9 +66,12 @@ func Load(file string) (config ProjectConfig, err error) {
 
 func fix(config ProjectConfig) ProjectConfig {
 	if len(config.Files) == 0 {
-		config.Files = []string{
-			"README.md",
-			"LICENSE.md",
+		config.Files = []string{}
+
+		for _, f := range []string{"README.md", "LICENCE.md", "LICENSE.md"} {
```

Comment:
nice, if you can't by any reason, let me know and I work on it =)

thanks for the contribution, by the way =D